### PR TITLE
Create SITE.yaml

### PR DESCRIPTION
**Pull request by plthomas1**
New Site Kent State University

### CODE REVIEW

**Comment by brianhlin** [comment]
@jstathas @ilandrum see new institution request here

File: topology/Kent State University/KSU-Research/SITE.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# LongName is the expanded name of the site
+LongName: Kent State University
+# Description is a brief description of your site
+Description: KSU Research Cluster
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_site_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID: <ID>
```
Comment (brianhlin) [inline topology/Kent State University/KSU-Research/SITE.yaml:8]:
```suggestion
ID: 10352
```

**Review by brianhlin — APPROVED** [review]
LGTM

**Comment by ilandrum** [comment]
I have reviewed the Kent State request as a new site and approve this request.

**Pull request closed without merging**